### ISSUE TITLE
libx265 has faulty replace_in_file

### ISSUE THREAD

**Issue by vadixidav** [bincrafters/community#362]
### Description of Problem, Request, or Question
I get this when pulling in libx265 to ffmpeg:
```
libx265/2.6@bincrafters/stable: Calling build()
libx265/2.6@bincrafters/stable:
libx265/2.6@bincrafters/stable: ERROR: Package '958e11b053522ba1213f524baa5fb80693d9ed53' build failed
libx265/2.6@bincrafters/stable: WARN: Build folder <I redacted this>
ERROR: libx265/2.6@bincrafters/stable: Error in build() method, line 35
        self.build_cmake()
while calling 'build_cmake', line 46
        '${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/x265.pdb')
        ConanException: replace_in_file didn't find pattern '${PROJECT_BINARY_DIR}/x265.pdb' in 'sources\source\CMakeLists.txt' file.
```

Looks like the replace_in_file requires the text to be there, but it isn't there in this case.

### Package Details (Include if Applicable)
* Package Name/Version: **libx265/2.6**
* Operating System: **Windows 10 Pro**
* Compiler+version: **VS 2015**

**Comment by vadixidav** [comment]
I will see if I can get to that tomorrow.

**Comment by vadixidav** [comment]
@SSE4 Yes, I can confirm that the issue happens in `libx265/2.6`, but it **does not** occur in `libx265/2.7`. `libx265/2.7` builds successfully for me.

**Comment by SSE4** [comment]
@vadixidav okay, so could you switch to 2.7 in your projects when?

**Comment by vadixidav** [comment]
It was due to a dependency on ffmpeg/3.4@bincrafters/stable. I could try overriding the dependency of ffmpeg with the newer version and hopefully it will build. I will let you know if ffmpeg 3.4 works with that x265 in a few days.

**Comment by SSE4** [comment]
FYI there is **ffmpeg/4.0@bincrafters/stable** recipe
